nominal/core/client.py

```diff
@@ -966,6 +976,64 @@ def search_events(
         )
         return list(self._iter_search_events(query))
 
+    def get_containerized_extractor(self, rid: str) -> ContainerizedExtractor:
+        return ContainerizedExtractor._from_conjure(
+            self._clients,
+            self._clients.containerized_extractors.get_containerized_extractor(self._clients.auth_header, rid),
+        )
+
+    def create_containerized_extractor(
+        self,
+        name: str,
+        *,
+        docker_image: DockerImageSource,
+        timestamp_column: str,
+        timestamp_type: ts._AnyTimestampType,
+        inputs: Sequence[FileExtractionInput] = (),
+        file_output_format: FileOutputFormat | None = None,
+        labels: Sequence[str] = (),
+        properties: Mapping[str, str] | None = None,
+        description: str | None = None,
+    ) -> ContainerizedExtractor:
+        workspace_rid = self._clients.workspace_rid
+        if workspace_rid is None:
+            workspace_rid = self.get_workspace().rid
+
+        req = ingest_api.RegisterContainerizedExtractorRequest(
+            image=docker_image._to_conjure(),
+            inputs=[file_input._to_conjure() for file_input in inputs],
+            labels=list(labels),
+            name=name,
+            properties={} if properties is None else {**properties},
+            timestamp_metadata=ingest_api.TimestampMetadata(
+                series_name=timestamp_column,
+                timestamp_type=_to_typed_timestamp_type(timestamp_type)._to_conjure_ingest_api(),
+            ),
+            workspace=workspace_rid,
+            description=description,
+            output_file_format=file_output_format._to_conjure() if file_output_format is not None else None,
+        )
+        resp = self._clients.containerized_extractors.register_containerized_extractor(self._clients.auth_header, req)
+        return self.get_containerized_extractor(resp.extractor_rid)
+
+    def search_containerized_extractors(
+        self,
+        search_text: str | None = None,
```

Comment:
all the search APIs use kw-only args

```suggestion
        self,
        *,
        search_text: str | None = None,
```